src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -40,6 +40,21 @@
     end
   end
 
+  describe '#word_break' do
+    it "continuously adds tag after N characters" do
+      expect(word_break("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", 10)).to \
+        eq("XXXXXXXXXX<wbr>XXXXXXXXXX<wbr>XXXXXXXXXX<wbr>XXXXXXXXXX")
+    end
+
+    it "adds no tag if string is shorter than N characters" do
+      expect(word_break("XX", 10)).to eq("XX")
```

Comment:
Yes, right! But the second parameter is in this case 10, so either you change it to 2 and test that still no tag has applied or you leave the parameter to 10 but add 10 characters as first parameter. 

And your assumption 
> NO tag was added if the given amount of characters is shorter than 
is wrong! Because it also doesn't add an character if the input string length == parameter!